### ISSUE TITLE
Images X and labels Y are not of same shape

### ISSUE THREAD

**Issue by nabilLearns** [nabilLearns/ChestXRay#6]
**Problem**
Image logits (X) are of shape [64, 15] whereas labels (Y) are of shape [64, 1, 15]. This does not allow for the computation of loss.

**TODO**
 - squeeze label shape
